<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1178</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="7" column="1" colspan="3">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="Time">
        <property name="text">
         <string>Time : </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="gradStartSpinbox"/>
      </item>
      <item>
       <widget class="QSpinBox" name="gradEndSpinBox"/>
      </item>
      <item>
       <widget class="QPushButton" name="getGradientButton">
        <property name="text">
         <string>Get Gradient</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="8">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="8">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="X">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>X1:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="X1spinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>X2:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="X2spinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="Y">
        <property name="text">
         <string>Y1:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="Y1spinBox"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Y2:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="Y2spinBox"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="8">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="8">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Gradient :</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="6">
     <widget class="QLabel" name="gradientValue">
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="Load">
      <property name="text">
       <string>Load</string>
      </property>
     </widget>
    </item>
    <item row="9" column="7">
     <widget class="QPushButton" name="quit">
      <property name="text">
       <string>Quit</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="8">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="progress">
        <property name="text">
         <string>Progress :</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="2">
     <widget class="QPushButton" name="SaveGradientGraph">
      <property name="text">
       <string>Save Gradient as CSV</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Video Width:</string>
      </property>
     </widget>
    </item>
    <item row="9" column="3">
     <widget class="QPushButton" name="saveBlur">
      <property name="text">
       <string>Save Blur as CSV</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="8">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="videoLocation">
        <property name="text">
         <string>Video Location </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="5">
     <widget class="QPushButton" name="saveAll">
      <property name="text">
       <string>Save All as CSV</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="videoWidthLabel">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QPushButton" name="saveSharp">
      <property name="text">
       <string>Save Sharp as CSV</string>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QLabel" name="videoHeightLabel">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="4">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Video Height:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0" colspan="8">
     <widget class="QToolBox" name="toolBox">
      <property name="contextMenuPolicy">
       <enum>Qt::NoContextMenu</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="DefaultPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1160</width>
         <height>283</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Default </string>
       </attribute>
       <widget class="QPushButton" name="selectROI">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>20</y>
          <width>211</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Display Frame and Select ROI</string>
        </property>
       </widget>
       <widget class="QPushButton" name="FluorscentIntensity">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>70</y>
          <width>391</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Get Fluorscent Intensity and Remove Blurry Frames</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Graphs">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>210</y>
          <width>97</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Graphs</string>
        </property>
       </widget>
       <widget class="QPushButton" name="selectGradientROI">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>160</y>
          <width>181</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Select ROI for Gradient</string>
        </property>
       </widget>
       <widget class="QLabel" name="totalNumFrames">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>20</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>490</x>
          <y>20</y>
          <width>16</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>/</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="userSelectFrameNumber">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>20</y>
          <width>101</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="AdvancePage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1160</width>
         <height>283</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Advance Settings</string>
       </attribute>
       <widget class="QLabel" name="KmeanLabel">
        <property name="geometry">
         <rect>
          <x>5</x>
          <y>30</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Kmean Iteration :</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="KmeanIterSpinBox">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>60</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="SobelKernalLabel">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>30</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Sobel Kernal size :</string>
        </property>
       </widget>
       <widget class="QLabel" name="BlurRemoveBoxLabel">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>100</y>
          <width>161</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Blur Removal PtP Box  :</string>
        </property>
       </widget>
       <widget class="QLabel" name="BlurRemoveBoxLabel2">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>30</y>
          <width>171</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Blur Remove Mean Box  :</string>
        </property>
       </widget>
       <widget class="QLabel" name="BlurRemovalLabel3">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>100</y>
          <width>171</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Blur Removal Tolerance :</string>
        </property>
       </widget>
       <widget class="QComboBox" name="SobelKernalSizeComboBox">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>30</y>
          <width>78</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>243</x>
          <y>0</y>
          <width>20</width>
          <height>291</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QLabel" name="KmeanLabel2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Kmean epsilon :</string>
        </property>
       </widget>
       <widget class="QComboBox" name="KmeansEpsilon">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>100</y>
          <width>78</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="BRMeanBoxComboBox">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>30</y>
          <width>78</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="BRPtPBoxComboBox">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>100</y>
          <width>78</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="BRtoleranceComboBox">
        <property name="geometry">
         <rect>
          <x>760</x>
          <y>100</y>
          <width>78</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="resetButtonAdvanceSet">
        <property name="geometry">
         <rect>
          <x>940</x>
          <y>70</y>
          <width>97</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Reset</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1178</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
